Error When Run Command `set` Without Parameter

# Error When Run Command `set` Without Parameter

Need handle this such an error

```bash
gostrap set          
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/piigyy/gostrap/cmd.glob..func4(0x97a380, {0x9b1ff8, 0x0, 0x0})
	/home/piigyy/go/pkg/mod/github.com/piigyy/gostrap@v1.0.0/cmd/set.go:31 +0xf5
github.com/spf13/cobra.(*Command).execute(0x97a380, {0x9b1ff8, 0x0, 0x0})
	/home/piigyy/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:860 +0x5f8
github.com/spf13/cobra.(*Command).ExecuteC(0x97a100)
	/home/piigyy/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/home/piigyy/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
github.com/piigyy/gostrap/cmd.Execute()
	/home/piigyy/go/pkg/mod/github.com/piigyy/gostrap@v1.0.0/cmd/root.go:37 +0x25
main.main()
	/home/piigyy/go/pkg/mod/github.com/piigyy/gostrap@v1.0.0/main.go:9 +0x17
```

--- a/cmd/set.go
+++ b/cmd/set.go
@@ -22,7 +22,7 @@ Current available config:
 - gomoduleplaceholder`,
 	Example: "gostrap set template https://github.com/golang-standards/project-layout",
 	Args: func(cmd *cobra.Command, args []string) error {
-		if len(args) > 2 {
+		if len(args) < 2 {
 			return fmt.Errorf("expected 2 arguments but found: %d", len(args))
 		}
 		return nil

